entrypoint.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/sh
 set -e
 cd /app
-./manage.py create_citext_extension
-./manage.py migrate
-#./manage.py runserver 0.0.0.0:8000
+uv run ./manage.py create_citext_extension
```

Comment:
Do we still need to create the citext extension? That should have been resolved in https://github.com/unioslo/mreg/pull/513